lib/hz2600/CallflowBuilder/Builder.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace CallflowBuilder;
+
+use CallflowBuilder\Node\AbstractNode; 
+
+class Builder {
+
+     /**
+     *
+     *
+     *
+     * @param array $numbers
+     * @param array $patterns
+     */
+    public function __construct(array $numbers = array(), array $patterns = array()){
+        if (!empty($numbers)){
+            $this->numbers = $numbers;
+        }
+        if (!empty($patterns)){ 
+            $this->patterns = $patterns;
+        } 
+    }
+
+     /**
+     *
+     *
+     *
+     * @param \CallflowBuilder\Node\AbstractNode $root_node
+     */
+    public function build(AbstractNode $root_node){
+        $this->flow = $root_node->build();  
+        return $this; 
+    }
+   
+     /**
+     *
```

Comment:
You are very welcome! Glad I could provide you with this information in such an effective manner!